### PR TITLE
updated configuring system users

### PR DESCRIPTION
**Pull request by AshVic**

### CODE REVIEW

File: docs/configure-system-users.md [hunk 1]
```diff
@@ -96,39 +96,76 @@ Follow the steps depending on which service or services you want to change the u
...
 ### Hive Migrator and Hive Migrator remote server
 
-1. Update the values for `HIVE_MIGRATOR_SERVER_USER` and `HIVE_MIGRATOR_SERVER_GROUP` in the configuration file for the intended service:
+1. Adjust the system user or group in the service configuration. Choose one of options below depending on your operating system.
 
-   ```text title="hivemigrator"
-   /etc/wandisco/hivemigrator/vars.sh
-   ```
+   1. **Linux distributions with systemd**
+      Create an override service file:
```
Comment (grainne-jennings) [inline docs/configure-system-users.md:102]: Need a new line here before 'Create'.

File: docs/configure-system-users.md [hunk 2]
```diff
@@ -96,39 +96,76 @@ Follow the steps depending on which service or services you want to change the u
...
 
 1. Change ownership of the related directories to the new system user/group:
 
-   ```text title="hivemigrator"
-   chown -R <user>:<group> /opt/wandisco/hivemigrator \
-   chown -R <user>:<group> /etc/wandisco/hivemigrator \
-   chown -R <user>:<group> /var/log/wandisco/hivemigrator \
-   chown -R <user>:<group> /var/run/hivemigrator
-   ```
+   1. **Linux distributions with systemd**
```
Comment (grainne-jennings) [inline docs/configure-system-users.md:130]: May be fine, but checking - is this section meant to be empty?

File: docs/configure-system-users.md [hunk 3]
```diff
@@ -96,39 +96,76 @@ Follow the steps depending on which service or services you want to change the u
...
-   chown -R <user>:<group> /opt/wandisco/hivemigrator-remote-server \
-   chown -R <user>:<group> /etc/wandisco/hivemigrator-remote-server \
-   chown -R <user>:<group> /var/log/wandisco/hivemigrator-remote-server \
-   chown -R <user>:<group> /var/run/hivemigrator-remote-server
-   ```
+
+   1. **Linux distributions without systemd**
+
+      ```text title="hivemigrator"
+      chown -R <user>:<group> /opt/wandisco/hivemigrator \
```
Comment (fi3ld) [inline docs/configure-system-users.md:136]: These commands are now grouped together in the public docs (advice from Tom M)

So they should be

```text title="livedata-migrator"
chown -R <user>:<group> /opt/wandisco/livedata-migrator /etc/wandisco/livedata-migrator /var/log/wandisco/livedata-migrator /var/run/livedata-migrator
```

```text title="livedata-ui"
chown -R <user>:<group> /opt/wandisco/ui /etc/wandisco/ui /var/log/wandisco/ui /var/run/livedata-ui /var/log/wandisco/audit/   
```

```text title="hivemigrator"
chown -R <user>:<group> /opt/wandisco/hivemigrator /etc/wandisco/hivemigrator /var/log/wandisco/hivemigrator /var/run/hivemigrator
```

```text title="hivemigrator-remote-server"
chown -R <user>:<group> /opt/wandisco/hivemigrator-remote-server /etc/wandisco/hivemigrator-remote-server /var/log/wandisco/hivemigrator-remote-server /var/run/livedata-hivemigrator-remote-server
```

File: docs/configure-system-users.md (hunk 2 above)
Comment (AshVic) [reply docs/configure-system-users.md]: Removed those headers as not relevant in that step

File: docs/configure-system-users.md (hunk 3 above)
Comment (AshVic) [reply docs/configure-system-users.md]: Added in thnx

**Pull request merged**